models/__init__.py

```diff
@@ -6,6 +6,23 @@
 session = Session()
 
 
+class ice_cost():
+    def __init__(self, uSkateUUID):
+        self.uSkateUUID = uSkateUUID
+
+    def ice_cost(self):
```

Comment:
If one method is expected and no state is required, then a class is not needed, and a function is preferred.

Suggestion here: `def ice_cost(uSkateUUID)`